projects/Mallard/src/components/front/items/helpers/text-block.tsx

```diff
@@ -58,12 +58,14 @@ const TextBlock = ({
     headline,
     style,
     byline,
+    type,
     monotone = false,
     inverted = false,
     ...sizes
 }: {
     kicker: string
     byline?: string
+    type?: string
```

Comment:
Would it be better to have `crossword` as a boolean? If we assume we only have this as the only variant.

Reply:
I wasn't sure  - I guess I thought that by simply passing through type it would allow for more variations if needed in future. But if you'd prefer to be more explicit am happy to change?

Reply:
Either is fine I think